src/pages/map/index.tsx

```diff
@@ -35,22 +52,24 @@ export default function Page() {
 
       <TabPanels>
         <TabPanel>
-          <LandmarkTabPanel latitude="个人成长" />
+          <LandmarkTabPanel latitude="个人成长" openDrawer={openDrawer}/>
         </TabPanel>
 
         <TabPanel>
-          <LandmarkTabPanel latitude="事业发展" />
+          <LandmarkTabPanel latitude="事业发展" openDrawer={openDrawer}/>
         </TabPanel>
 
         <TabPanel>
-          <LandmarkTabPanel latitude="社会责任" />
+          <LandmarkTabPanel latitude="社会责任" openDrawer={openDrawer}/>
         </TabPanel>
       </TabPanels>
     </TabsWithUrlParam>
-  );
+    {<LandmarkDrawer isOpen={isOpen} onClose={closeDrawer} landmark={selectedLandmark} />}
+  </>;
 }
 
-const LandmarkTabPanel = ({ latitude }: { latitude: Latitude }) => {
+const LandmarkTabPanel = ({ latitude, openDrawer }: { 
```

Comment:
rename `openDrawer` to `selectLandmark`. TabPanel or none of its child components should have knowledge about the drawer.